src/components/ViewQuestionComponent.tsx

```diff
@@ -22,11 +24,17 @@ export const ViewQuestionComponent = ({
       <Heading size={900} marginTop="default">
         {question.text}
       </Heading>
+      {question.embedsType === "none" ? (
+        ""
+      ) : question.embedsType === "photo" ? (
+        <img src={question.embeds}></img>
+      ) : (
+        <ReactPlayer url={question.embeds} />
+      )}
```

Comment:
If you return `null` or `undefined` nothing will be rendered - the idiomatic way is to return null instead of an empty string.